anime-list-master.xml

```diff
@@ -12463,6 +12463,9 @@
   </anime>
   <anime anidbid="4224" tvdbid="113571" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Soul Link</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0" start="1" end="4" offset="1">;1-1+2;</mapping>
```

Comment:
You're mapping sp01 twice here. Absolute usually takes precedence over offset but it's best to avoid conflicts if possible.